tests/integration_tests/saas/test_zendesk_task.py

```diff
@@ -169,3 +174,173 @@ def test_zendesk_access_request_task(
+            "active",
+            "shared",
+            "shared_agent",
+            "last_login_at",
+            "two_factor_auth_enabled",
+            "signature",
+            "details",
+            "notes",
+            "role_type",
+            "custom_role_id",
+            "moderator",
+            "ticket_restriction",
+            "only_private_comments",
+            "restricted_agent",
+            "suspended",
+            "default_group_id",
+            "report_csv",
+        ],
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:tickets"],
+        min_size=1,
+        keys=[
+            "url",
+            "id",
+            "external_id",
+            "via",
+            "created_at",
+            "updated_at",
+            "type",
+            "subject",
+            "raw_subject",
+            "description",
+            "priority",
+            "status",
+            "recipient",
+            "requester_id",
+            "submitter_id",
+            "assignee_id",
+            "organization_id",
+            "group_id",
+            "collaborator_ids",
+            "follower_ids",
+            "email_cc_ids",
+            "forum_topic_id",
+            "problem_id",
+            "has_incidents",
+            "is_public",
+            "due_at",
+            "tags",
+            "custom_fields",
+            "satisfaction_rating",
+            "sharing_agreement_ids",
+            "fields",
+            "followup_ids",
+            "brand_id",
+            "allow_channelback",
+            "allow_attachments",
+        ],
+    )
+
+    x = graph_task.run_erasure(
+        privacy_request,
+        erasure_policy_string_rewrite,
+        graph,
+        [zendesk_connection_config],
+        {"email": zendesk_erasure_identity_email},
+        get_cached_data_for_erasures(privacy_request.id),
+    )
+
+    assert x == {
+        f"{dataset_name}:users": 1,
+        f"{dataset_name}:user_identities": 0,
+        f"{dataset_name}:tickets": 1,
+        f"{dataset_name}:ticket_comments": 0,
+    }
+
+    zendesk_secrets = zendesk_connection_config.secrets
+    base_url = f"https://{zendesk_secrets['domain']}"
+    headers = {
+        "Content-Type": "application/json",
+        "Authorization": "Basic {}".format(
+            base64.b64encode(
+                bytes(
+                    f"{zendesk_secrets['username']}:{zendesk_secrets['api_key']}",
+                    "utf-8",
+                )
+            ).decode("ascii")
+        ),
+    }
+
+    # user
+    response = requests.get(
+        url=f"{base_url}/v2/users",
+        headers=headers,
+        params={"email": zendesk_erasure_identity_email},
+    )
+    # Since user is deleted, it won't be available so response is 404
+    assert response.status_code == 404
+
+    tickets = x[f"{dataset_name}:tickets"]
+    ticket_id = v[f"{dataset_name}:tickets"][0]["id"]
+    response = requests.get(
+        url=f"{base_url}/v2/tickets/f{ticket_id}.json",
```

Comment:
Looks like you have an extra character `f` before the `{ticket_id}`, this is most likely returning a 404 because it's an invalid path and not because it's actually checking the ticket. Make sure you have the correct path.